docs/sources/panels/visualizations/gauge-panel.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Gauge panel"
+description = "Gauge panel docs"
+keywords = ["grafana", "gauge", "gauge panel"]
+type = "docs"
+aliases = ["/docs/grafana/latest/features/panels/gauge/"]
+weight = 300
+draft = "true"
++++
+
+# Gauge panel
+
+Gauge is a single value panel that can repeat a gauge for every series, column or row.
+
+{{< docs-imagebox img="/img/docs/v66/gauge_panel_cover.png" max-width="1025px" >}}
+
+## Data and field options
+
+Gauge visualizations allow you to apply:
+- Data transformations
+- Field settings
+- Field overrides
+- [Thresholds]({{< relref "../thresholds.md">}})
+- Value mappings
+
+## Display options
+
+Use the following options to refine your visualization:
+
+- **Show -** Choose how Grafana displays your data.
+  - **Calculate -** Show a calculated value based on all rows.
```

Comment:
Would it be a good idea to add link to calculations list here?

Reply:
Yes!